tools/infrasys/cloudformation-templates/s3_setup.yml

```
@@ -0,0 +1,25 @@
+Resources:
+  TUFRepoBucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain # if we want to delete, chnge this 
```

I think we'd always stick with retaining the bucket. So no need for the comment.
```suggestion
    DeletionPolicy: Retain
```

---

Using Delete Retain: when the user tries to delete the S3 stack it says "successful deletion" but leaves the bucket there.
Without Delete Retain: when the user tries to delete the S3 stack it tries to delete and says "this bucket is full, first empty it"
I think the second message might be nicer? So if users delete their S3 stack, they are not surprised when they see their bucket in the S3 list
The other option is that we document that deleting the S3 stack will not actually clean up its resources (but this might be missed)

---

It's pretty standard to always retain buckets when stacks are deleted to prevent any possible data loss (including configuration). Since it's highly possible there are downstream services dependent on said bucket.
We can just log in Infrasys something right before to say the bucket will be retained.